bot/commands/cmd_verify.go

```diff
@@ -270,7 +273,7 @@ func (cm *CommandManager) WarnUserVerificationState(channelid string, verificati
 	case types.VerificationStateAwaitProfileURL:
 		stateMessage = "Your verification is currently awaiting a profile URL or profile ID."
 	case types.VerificationStateAwaitConfirmation:
-		stateMessage = "Your verification is currently awaiting you to post the verification code on your Profile Bio, once you've done that reply with either 'done' or 'cancel'"
+		stateMessage = "Your re-verification is currently awaiting you to add your discord ID to your profile, once you've done that reply with either 'done' or 'cancel'"
```

Comment:
Don't need to call it `re-verification` tbh, imagine it from a new user perspective.

Reply:
Also, again, commands with backticks instead of '

Reply:
Must have been a typo for when I was doing the reverify command.